### DISCUSSION TITLE
Let's start gathering some ideas for UI improvements

### DISCUSSION THREAD

**Discussion by tobybellwood** [amazeeio/lagoon#2637] [Ideas]
Let's collect some ideas for UI improvements here.

We'll try to look at whatever we can in the next round(s) of UI work.  We'll also endeavour to show some prototypes of what we're working on (and occasionally call out for people to test/walkthrough)

**Comment by brett-sargeant** [comment]
It would be really nice if we could choose the order of the projects on https://dashboard.govcms.gov.au/projects for example alphabetically or recently modified or added.  It would also be nice if we could filter out deactivated ones.

**Reply by tobybellwood** [reply]
Thanks, @brett-sargeant - for the ones that are "deactivated", do they have no environments?

**Reply by brett-sargeant** [reply]
yes, no environments attached to them - sites that have been decommissioned. I guess they are still there for archival purposes.

**Comment by campbell-t** [comment]
The ability to copy/move a database from one environment to another, within the UI, would be nice.

**Reply by tobybellwood** [reply]
Thanks @campbell-t - we're currently working on a much more flexible task system, and one of those tasks is OOTB db syncing (all db services, all platforms, and including <> local).

We've got a CLI tool that'll form the base of it at https://github.com/amazeeio/lagoon-sync that's ready for use, but having it tied into the UI would be :100:

**Comment by tobybellwood** [comment]
From @seanhamlin  https://github.com/amazeeio/lagoon/issues/2595 - Lagoon UI: Add pagination to the backup list 

At the moment, for projects that have existed for a long time, it is not uncommon to have many hundreds, if not thousands of backups to be listed on the environment backup page. Here is an example of a project with a lot of backups:

```
document.getElementsByClassName('data-row').length
1438
```

I would say the 99% use case is to get a recent backup, and not to scroll down the page and hunt for the backup.
Feature request

Can we add pagination to the backup list screen, and show perhaps the last 7 days worth of backups, or the most recent 20 or something?

Related to #1382

**Comment by grahamethompson** [comment]
Deploy badges for the state of the latest deployment would be nice on the project's homepage. They could link straight to that deploy's logs and perhaps have some kind of tooltip with the build time and date.

![image](https://user-images.githubusercontent.com/16107883/117895847-966d8b00-b302-11eb-9e96-f62dba50f389.png)
